homeassistant/components/sensor/bom.py

```diff
@@ -164,19 +163,19 @@ def _build_url(self):
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Get the latest data from BOM."""
-        if self._lastupdate != 0 and \
-            ((datetime.datetime.now() - self._lastupdate) <
-             datetime.timedelta(minutes=35)):
+        curr_time = datetime.datetime.now(datetime.timezone.utc)
+        if self._lastupdate is not None and \
+            ((curr_time - self._lastupdate) < datetime.timedelta(minutes=30)):
```

Comment:
The tests have to pass before we will merge: https://travis-ci.org/home-assistant/home-assistant/jobs/188492332
So you have fix the indention.